.eslintrc.json

```diff
@@ -1,10 +1,13 @@
 {
   "env": {
+    "node": true,
     "browser": true,
-    "es6": true
+    "es6": true,
+    "mocha": true
   },
 
   "extends": [
+    "airbnb-base",
```

Comment:
As i mentioned before dont use base. Did you take a look at just eslint-config airbnb? Use airbnb, its closer to the style you want.